confd/vars/main.yaml

```diff
@@ -24,6 +24,7 @@ kilda_victoriametrics_host: "victoriametrics.pendev"
 kilda_victoriametrics_write_port: "4242"
 kilda_victoriametrics_read_port: "8428"
 kilda_victoriametrics_path: ""
+kilda_victoriametrics_read_path: ""
```

Comment:
can `kilda_victoriametrics_path` be different with `kilda_victoriametrics_read_path` ?

Reply:
It should be different if not empty